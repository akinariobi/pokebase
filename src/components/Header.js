import React from "react";

function Header() {
  return (
    <div className="container Header__container">
      <h1>Pok√©base</h1>
    </div>
  );
}

export default Header;
