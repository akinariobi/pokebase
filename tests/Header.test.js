import React from "react";
import Header from "../src/components/Header";
import renderer from "react-test-renderer";

describe("Header", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Header />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
