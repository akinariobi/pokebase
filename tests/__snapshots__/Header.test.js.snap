// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot renders 1`] = `
<div
  className="container Header__container"
>
  <h1>
    Pok√©base
  </h1>
</div>
`;
